src/index.html

```diff
@@ -16,17 +19,33 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-bar__body">
+    <form
+      class="search-bar__form search-bar__form--big"
+      action="http://localhost:8080/"
+      data-qa="big"
+    >
+      <input
+        class="search-bar__input search-bar__input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        autocomplete="on"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar__form search-bar__form--small"
+      action="http://localhost:8080/"
+      data-qa="small"
```

Comment:
The indentation is not consistent. Ensure that the attributes of the `form` tag are properly indented with 2 spaces.